OneCommunityGlobal/HighestGoodNetworkApp
# MonthlyEffort and OwnerMessage tests

# Description


<google-sheets-html-origin>

src/components | file | has unit tests
-- | -- | --



<google-sheets-html-origin>

MonthlyEffort | MonthlyEffort.jsx | Yes. Need to move the file under the test folder
-- | -- | --


<google-sheets-html-origin>

OwnerMessage | OwnerMessage.jsx | No
-- | -- | --






https://docs.google.com/spreadsheets/d/1t26Y9h3yRJAfvaWbveRzOz28plXWNgbe3A8Ceiqwyu4/edit#gid=0


## Main changes explained:
MonthlyEffort tests moved and OwnerMessage tests created

## How to test:
1. check into the current branch and git pull
2. run the command `npm test -- --testPathPattern=src/__tests__/OwnerMessage.test.js` 
3. Check if the tests are passing
4. run the command `npm test -- --testPathPattern=src/__tests__/MonthlyEffort.test.js`
5. Check if the tests are passing
